Makefile

```diff
@@ -52,15 +52,17 @@ vet: ## Run go vet
 
 check: fmtcheck vet lint build test ## Pre-flight checks before creating PR
 
-run: broker | $(KUBERNETES_FILES) ## Run the broker locally, configure via etc/generated_local_development.yaml
+run: broker
 	@./scripts/run_local.sh ${BROKER_CONFIG}
 
-$(KUBERNETES_FILES):
+# NOTE: Must be explicitly run if you expect to be doing local development
+# Basically brings down the broker pod and extracts token/cert files to
+# /var/run/secrets/kubernetes.io/serviceaccount
+# Resetting a catasb cluster WILL generate new certs, so you will have to
+# run prep-local again to export the new certs.
+prep-local:
```

Comment:
Does this need to be in `.PHONY` list?

Reply:
Add a comment after prep-local so that it shows up in the help.

```
prep-local: ## Prepares the local dev environment
```

Then test it with 
```
$ make help
Usage: make <OPTIONS> ... <TARGETS>

Available targets are:

vendor               Install or update project dependencies
broker               Build the broker
build                Build binary from source
lint                 Run golint
fmt                  Run go fmt
fmtcheck             Check go formatting
test                 Run unit tests
...
```

Reply:
That's a nifty trick.